manage.py

```diff
@@ -118,7 +118,7 @@ def print_flush(msg):
     print_every = 1000
     print_flush("Adding waterpoints. Please be patient.")
 
-    with open(filename) as f:
+    with open(filename, 'rU') as f:
```

Comment:
What are the line terminators for this file? Not opposed to this change, just curious.